<launch>
  <!-- <arg name="algorithm_to_run" default="BSG"/> -->
  <arg name="algorithm_to_run" default="RBG"/>
  <arg name="replanning_frequency" default="10"/>

  <!-- Drone 1 -->
  <node pkg="multi_target_tracking" type="multi_target_tracking_coord" name="coordination_debug">
    <param name="drone_name" value="Drone1"/>
    <param name="robot_index_ID" type="int" value="1"/>
    <param name="number_of_pursuers" value="2"/>
    <param name="number_of_evaders" value="2"/>
    <param name="algorithm_to_run" value="$(arg algorithm_to_run)"/>
    <param name="replanning_frequency" value="$(arg replanning_frequency)"/>

    <param name="camera_name" value="front_center"/>
    <param name="image_topic" value="/camera_1/image"/>
    <param name="communication_range" type="double" value="150"/>
  </node>

</launch>
